themes/src/main/resources/theme/base/admin/resources/js/app.js

```diff
@@ -214,8 +214,8 @@ module.config([ '$routeProvider', function($routeProvider) {
                 realm : function(RealmLoader) {
                     return RealmLoader();
                 },
-                serverInfo : function(ServerInfo) {
-                    return ServerInfo.delay;
+                serverInfo : function(ServerInfoLoader) {
+                    return ServerInfoLoader();
```

Comment:
This fixes an untracked issue when the Client Policy tab was not visible at the Login Settings page. Not sure if this change is valid, though. The delay was introduced [here](https://github.com/keycloak/keycloak/commit/d1964c20abbf93731781e230747b0155224c597d#diff-a6199d48042845af1f16f971eb297365d4455dc3d47c4b511b1e0f2e5338eabcR85-R86) and I've got no clue why. This change doesn't seem to break anything but I'll run the UI tests just to be sure and I'll add the results here once I have them.

Reply:
Good catch. Agree will be better to doublecheck by running UI tests

Reply:
The test results are here: https://keycloak-jenkins.com/job/universal-test-pipeline-server/419/

A few tests failed but non of them seem to be related to this change. I tried the failed tests locally and compared to master and there seem to be no regressions. I think we're good to go.